src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body,
+html {
+  margin-top: 0;
+}
+
+.form__first,
+.form__second {
+  position: relative;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background: url(./images/Search.svg) no-repeat;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form__first {
+  top: 20px;
+  padding-left: 74px;
+  background-position: 25px;
+  background-size: 20px;
+}
+
+.form__second {
+  top: 40px;
+  padding-left: 34px;
+  background-position: 12px;
+  background-size: 11px;
+}
```

Comment:
to much duplications, and you don't need to position everything using `position:relative` that's one of a reason why you can't pass  tests

Reply:
Em... In this programm I use one [position](position:relative))
Duplications need for each search bar. I cant detete this.

Reply:
take a look over point 2 of [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
there is answer how to manage it to avoid duplications
and do not try to solve problem in PR chats - if smth is unclear or you can manage smth - write directly in `fe_chat0` and tag mentors - we will help you and it will be much faster

Reply:
Thanks for open answer) 
In chat, I got answers very long (I have question 2 month ago, and mentors still can't answer).